roles/confluent.common/tasks/CentOS.yml

```diff
@@ -0,0 +1 @@
+RedHat.yml
```

Comment:
is this a symlink? unfortunately, some of our customers did some funky stuff with moving the source code around and it messed up symlinks when i added them before, so I think we are trying not to rely on them (bc we can't trust ppl's git skills)

To avoid symlinks, you should be able to make this a task file that just calls the red hat tasks file

Reply:
ah too bad - but ok I will change that :)